ci-operator/config/3scale-qe/3scale-deploy/3scale-qe-3scale-deploy-main__3scale-amp-ocp4.13-lp-interop.yaml

```diff
@@ -40,6 +40,18 @@ tests:
       BASE_DOMAIN: cspilp.interop.ccitredhat.com
       DEPL_BUCKET_NAME: 3scale-apimanager-s3-bucket
       DEPL_PROJECT_NAME: threescale
+      FIREWATCH_CONFIG: |
+        [
+          {"step": "ipi-*", "failure_type": "all", "classification": "Infrastructure Provisioning - Cluster", "jira_project": "LPTOCPCI"},
+          {"step": "install-operators", "failure_type": "all", "classification": "Infrastructure Provisioning - Operators", "jira_project": "LPTOCPCI"},
```

Comment:
I take this to be a product install failure. I am debating whether this is the kind of failure that should be posted in our project or PQEs project. I think this is something that I would like to discuss broadly with the team.

Reply:
updating "jira_project": "THREESCALE" for failures while installing operators